fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java

```diff
@@ -74,15 +74,11 @@ private ConventionalItemTags() {
 	public static final TagKey<Item> NUGGETS = register("nuggets");
 	public static final TagKey<Item> ORES = register("ores");
 	public static final TagKey<Item> RAW_MATERIALS = register("raw_materials");
-	public static final TagKey<Item> RAW_BLOCKS = register("raw_blocks");
```

Comment:
Breaking change, im not super keen on making more breakig changes like this, it should be deprecated.

Reply:
just adding the deprecated tag and not datagen the files?

Reply:
Sounds good, I added the tags back with their datagen. But I moved them to the bottom of ConventialItemTags, put a deprecated tag on them, and javadoc ont hem saying they are redundant with what replacement to use. Plus the console will also warn about using them if detected as well. That might be good enough to get people off the tags to replacements without breaking